### DISCUSSION TITLE
Discord Channel

### DISCUSSION THREAD

**Discussion by ashwin-ned** [erodola/DLAI-s2-2022#9] [Q&A] [answered]
Hi, 

Is the discord channel for DLAI already active? 
if so could you share the link to join somewhere?

**Comment by erodola** [comment, marked as answer]
Yes, it was published on February 23, you can find it in the slides of the [first lecture](https://github.com/erodola/DLAI-s2-2022/raw/main/01_intro/01-intro.pdf).
